src/capabilities.rs

```diff
@@ -194,6 +195,28 @@ impl SpecNewSessionParameters {
         Ok(())
     }
 
+    fn validate_no_proxy(value: &Json) -> WebDriverResult<()> {
+        match value.as_array() {
+            Some(hosts) => {
+                for host in hosts {
+                    match host.as_string() {
+                        Some(_) => {},
+                        None => return Err(WebDriverError::new(
+                            ErrorStatus::InvalidArgument,
+                            format!("{} was not a string", value)
```

Comment:
The spec says `A List containing any number of any of domains, IPv4 addresses, or IPv6 addresses.`, but when you check https://developer.mozilla.org/en-US/docs/No_Proxy_For_configuration this is way more than those. As such we should only validate that entries in that list are of type `String`.

Reply:
Then I think the specification needs to be updated.

Reply:
Which will be done via https://github.com/w3c/webdriver/issues/1068.